app/Account.php

```diff
@@ -357,7 +401,7 @@ public function getNextBillingDate()
      */
     public function hasLimitations()
     {
-        if ($this->has_access_to_paid_version_for_free) {
+        if ($this->has_access_to_paid_version_for_free == 0) {
```

Comment:
Idem : "return ! $this->has_access_to_paid_version_for_free == 0"

Reply:
Done